subsystem/infra_env_test.go

```diff
@@ -197,13 +199,94 @@ var _ = Describe("Infra_Env", func() {
 		u := res.Payload.URL
 		Expect(u).NotTo(BeNil())
 
+		Expect(*u).ToNot(ContainSubstring("boot_control"))
```

Comment:
This seems to be failing the subsystem test.

I'm not sure how important it is to ensure this isn't present in the ipxe-script case though. (my original concern was just about other files at this endpoint).

@avishayt do you think it would be particularly bad for this url to change after an upgrade even though the contents would be the same?

It's not the iso url so we won't cause a host reboot or anything. :man_shrugging:

Reply:
Now it doesn't fail